### PR TITLE
#330 Fix company permissions & test

### PR DESCRIPTION
**Pull request by vryazanov**
#330 
* Fix passing args in base class responsible for permissions
* fix company permissions for owner role

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#426](https://github.com/timelesslounge/timelessis/issues/426) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#426](https://github.com/timelesslounge/timelessis/issues/426) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/e618e6a1-19cd-4b2d-b9f9-8dfbb723a637); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/access_control/owner_privileges.py [hunk 1]
```diff
@@ -33,10 +33,15 @@ def __employee_access(method=None, *args, **kwargs):
...
+def __company_access(method=None, *args, **kwargs):
+    user, company_id = flask.g.get("user"), kwargs.get("company_id")
+    return user.company_id == company_id
+
+
 __resources = {
     "location": __location_access,
     "employee": __employee_access,
-    "companies": __employee_access,
+    "companies": __company_access,
```
Comment (vladarefiev) [inline timeless/access_control/owner_privileges.py]: @vryazanov according to others, let's do `"company": __company_access,`

**Comment by vladarefiev** [comment]
@vryazanov everything is good, only one thing I want to ask you to change.

**Comment by vryazanov** [comment]
@vladarefiev  done, pls have a look

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 13 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#426](https://github.com/timelesslounge/timelessis/issues/426) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev  According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**